Building of elyra docker image fails on current master

tests/integration/pipeline.ts 67ms
tests/integration/pythoneditor.ts 47ms
tests/integration/submitbutton.ts 35ms
tests/integration/toc.ts 41ms
tests/plugins/index.js 31ms
tests/tsconfig.json 38ms
tests/webpack.config.js 139ms
tsconfig-base.json 32ms
Done in 12.00s.
yarn run eslint
yarn run v1.22.4
$ eslint . --fix --ignore-path .gitignore --ext .ts,.tsx,.js
Done in 6.51s.
export PATH=$(pwd)/node_modules/.bin:$PATH && lerna run build
lerna notice cli v3.22.1
lerna info Executing command in 8 packages: "yarn run build"
lerna ERR! yarn run build exited 139 in '@elyra/ui-components'
lerna ERR! yarn run build stdout:
yarn run v1.22.4
$ tsc
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build stderr:
Segmentation fault (core dumped)
error Command failed with exit code 139.
lerna ERR! yarn run build exited 139 in '@elyra/ui-components'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
Makefile:83: recipe for target 'build-ui' failed
make: *** [build-ui] Error 139
The command '/bin/sh -c python -m pip install --upgrade pip &&     python -m pip install setuptools pandas --ignore-installed --upgrade &&     echo "scripts-prepend-node-path=true" >> /home/jovyan/.npmrc &&     echo "prefix=/home/jovyan/.npm-global" >> /home/jovyan/.npmrc &&     mkdir -p /home/jovyan/.npm-global &&     npm install -g yarn &&     npm install -g npm &&     cd /tmp && git clone https://github.com/elyra-ai/elyra.git &&     cd /tmp/elyra && git checkout tags/v1.0.0 -b v1.0.0 && make install' returned a non-zero code: 2

classic.yarnpkg.comclassic.yarnpkg.com
Yarn
Fast, reliable, and secure dependency management.

--- a/docs/source/developer_guide/development-workflow.md
+++ b/docs/source/developer_guide/development-workflow.md
@@ -90,7 +90,7 @@ To see your changes just refresh JupyterLab in your browser.
 So when making changes to application you will need to stop and restart `jupyter lab --watch` and
 not just refresh your browser.
 
-### Building Elyra docker
+### Building the Docker Image
 
 Build Elyra docker image locally with:
 

--- a/docs/source/developer_guide/development-workflow.md
+++ b/docs/source/developer_guide/development-workflow.md
@@ -27,7 +27,32 @@ directory.
 
 * [Yarn](https://yarnpkg.com/lang/en/docs/install) 
 
-#### Installation
+#### Build & Installation
+
+Elyra uses make to automate some of the development workflow tasks.
+
+Issuing a make command with no task specified will provide a list of the currently supported tasks.
+
+```bash
+$ make
+
+build-server                   Build backend
+build-ui                       Build packages
+clean                          Make a clean source tree and uninstall extensions
+docker-image                   Build docker image
+docs                           Build docs
+install-server                 Install backend
+install                        Build and install
+lint                           Run linters
+release                        Build wheel file for release
+test-server                    Run unit tests
+test-ui                        Run frontend tests
+test                           Run all tests
+validate-runtime-images        Validates delivered runtime-images meet minimum criteria
+watch                          Watch packages. For use alongside jupyter lab --watch
+```
+
+You can build and install all Elyra packages with:
 
 ```bash
 make clean install
@@ -64,3 +89,11 @@ To see your changes just refresh JupyterLab in your browser.
 > NOTE: JupyterLab watch mode will not pick up changes in package dependencies like `application`.
 So when making changes to application you will need to stop and restart `jupyter lab --watch` and
 not just refresh your browser.
+
+### Building Elyra docker
+
+Build Elyra docker image locally with:
+
+```bash
+make docker-image
+```

--- a/docs/source/developer_guide/development-workflow.md
+++ b/docs/source/developer_guide/development-workflow.md
@@ -92,7 +92,7 @@ not just refresh your browser.
 
 ### Building the Docker Image
 
-Build Elyra docker image locally with:
+Elyra's docker image can be built using:
 
 ```bash
 make docker-image

--- a/docs/source/developer_guide/development-workflow.md
+++ b/docs/source/developer_guide/development-workflow.md
@@ -97,3 +97,5 @@ Elyra's docker image can be built using:
 ```bash
 make docker-image
 ```
+
+Elyra official docker images are available at the [Elyra organization in dockerhub](https://hub.docker.com/r/elyra/elyra).

